build/coffee-esm.mjs

```diff
@@ -3,9 +3,7 @@
 import { readFile } from 'fs/promises';
 import { pathToFileURL, fileURLToPath } from 'url';
 
-import CoffeeScript, { compile } from "coffeescript";
-// Handle cjs .coffee files
-CoffeeScript.register()
```

Comment:
I think it's nice to keep this registration in case others want to use this as a CoffeeScript ESM handler. So I lean toward keeping it, but it's also OK to remove it and leave this for a CoffeeScript issue (actually https://github.com/jashkenas/coffeescript/pull/5447).